src/developer/web-api/analytics.md

```diff
@@ -1846,6 +1846,180 @@ For example, to retrieve a list of enrollments from the "WHO RMNCH Tracker" prog
 
 The API supports using program indicators which are not associated to the "main" program (that is the program ID specified after `/query/`).
 
+## Dimensions { #webapi_dimensions }
+
+Four resources allow to easily retrieve data dimensions:
```

Comment:
Sometimes the term `resource` is used, and sometimes the term `endpoint` is used.
I would use only one term (`endpoint` in those cases) to make it easier for the users. But this is my personal choice. I leave it to you to decide.

Reply:
yeah you're right, I used resource because in this file previous endpoints are reported as "resource". But I can change it.